src/main/java/com/onarandombox/MultiverseCore/commands/ListCommand.java

```diff
@@ -43,52 +41,52 @@ public ListCommand(MultiverseCore plugin) {
 
     @Override
     public void runCommand(CommandSender sender, List<String> args) {
-        ContentFilter filter = ContentFilter.DEFAULT;
+        ContentFilter filter = DefaultContentFilter.INSTANCE;
```

Comment:
I think convention might say use a getter method for this.

Reply:
Yea use the singleton design pattern